server/npg/porch/models/permission.py

```diff
@@ -24,11 +24,18 @@
 
 from npg.porch.models.pipeline import Pipeline
 
+
+class PermissionValidationException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
```

Comment:
Not a very useful constructor. I would have thought that inheriting from Exception would allow you to skip `__init__` entirely

Reply:
Thank you, fixed